temporalcli/commands.activity.go

```diff
@@ -208,6 +208,10 @@ func (c *TemporalActivityPauseCommand) run(cctx *CommandContext, args []string)
 		Identity: c.Identity,
 	}
 
+	if c.ActivityType != "" && c.ActivityId != "" {
```

Comment:
Can also just toss a `if c.ActivityId != ""` inside the `if c.ActivityType != ""` body below, but this is fine too